test/vulnz-check/mirror_images.py

```diff
@@ -0,0 +1,71 @@
+# %%
+from typing import NamedTuple
+import os
+
+import kfp
+from kfp import dsl
+from kfp.components import func_to_container_op, InputPath, OutputPath
+
+# %%
+# Mirror Image
+
+
+def mirror_image(
```

Comment:
Maybe this component is easier to define in YAML form, since it just calls a command-line program.